DESCRIPTION

```diff
@@ -105,13 +105,14 @@ Suggests:
     ggplot2,
     reactlog (>= 1.0.0),
     magrittr,
-    shinytest (>= 1.4.0.9003),
     yaml,
     future,
     dygraphs,
     ragg,
     showtext,
     sass
+Suggests/future:
+    shinytest
```

Comment:
This should be shinytest2? (I'm not sure what `Suggests/future` is for)

Reply:
That's twice now! 🤦‍♂️

> I'm not sure what `Suggests/future` is for

For a followup release, we can add in the dependency of `{shinytest2}` to be safe, but it is nice to avoid the chicken / egg / CRAN problem until `{shinytest2}` is on CRAN.

Reply:
```suggestion
    shinytest2
```

Reply:
Added a TODO in the test file